lib/deploy/strategy.js

```diff
@@ -0,0 +1,8 @@
+const strategies = {
```

Comment:
@pavelloz Nice one :)

Reply:
Its just temp, when we switch to direct assets upload there will be only one way, so I didn't want to waste time on technicalities (hence not DRY inside of strategies) ;)